tools/check-c.hs

```diff
@@ -42,7 +42,7 @@ main = do
     let (opts, rest) = partition (isPrefixOf "--") args
     let (cppOpts, files) = partition (isPrefixOf "-") rest
     let cc = Maybe.fromMaybe "clang" $ getFlag "--cc=" opts
-    let sysInclude = "/src/workspace/hs-tokstyle/include"
+    let sysInclude = Maybe.fromMaybe "src/workspace/hs-tokstyle/include" $ getFlag "--include=" opts
```

Comment:
Was removing the leading `/` intentional?

Reply:
Nope